cloud-functions/comparer-function/main.py

```diff
@@ -137,15 +139,33 @@ def publish_message(project_id, topic_name, message):
     future = publisher.publish(topic_name, b'hello from compare func')
     future.result()
 
+# define the function to compute Mean Squared Error between two images
+def meanSquaredError(img1, img2):
+   height, width = img1.shape
+   diff = cv2.subtract(img1, img2)
+   err = np.sum(diff**2)
+   mse = err/(float(height*width))
+   return mse, diff
+
+def compare(image_a, image_b, imageMask):
```

Comment:
- refactor to `image_mask` to follow convention

Reply:
done